core/iwasm/common/arch/invokeNative_mingw_x64.s

```diff
@@ -55,3 +55,6 @@ cycle_end:
     call *%r10
     leave
     ret
+#if defined(__linux__) && defined(__ELF__)
+.section .note.GNU-stack,"",%progbits
+#endif
```

Comment:
i suspect these mingw files are for windows.

Reply:
reverted the mingw files.  It shouldn't have an effect in that case of compiling for windows ... but still, it could cause confusion.